src/components/BoardUsers/BoardUser.tsx

```diff
@@ -0,0 +1,23 @@
+import './BoardUser.scss';
+import {getInitials} from 'constants/Name'
+export interface BoardUserProps {
+  id: string;
+  name: string;
+  avatar?: string;
+}
+
+const BoardUser = ({ name, avatar }: BoardUserProps) => {
+
+  return (
+    <li>
+      <div className="user">
```

Comment:
I think this `div` can be removed and the styles can be applied on the parent `li` element. Also I like to name the CSS classes just like the components name, so that is easier to find on debug of the HTML code. In this case it would be `board-user` or `user-avatar` as I suggested a rename of the component. Whatever you prefer :)

Reply:
done. I liked `user-avatar` more :)